recipe/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "recipe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node --inspect=5858 -r ts-node/register ./src/index.ts",
```

Comment:
@no-or to use the config, install env-cmd like the other services and put `env-cmd -f ../config/dev.env` in here at the beginning.